otlp/traces_test.go

```diff
@@ -182,6 +166,22 @@ func TestTranslateGrpcTraceRequest(t *testing.T) {
 			assert.Equal(t, "link", ev.Attributes["meta.annotation_type"])
 			assert.Equal(t, "span_link_attr_val", ev.Attributes["span_link_attr"])
 			assert.Equal(t, "resource_attr_val", ev.Attributes["resource_attr"])
+
+			// span
+			ev = events[2]
```

Comment:
Can we update/add tests that verify the error span event attributes are added to the honeycomb event for the span?

Reply:
I've added a test for that.